Packs/Arcanna/Scripts/PrepareArcannaRawJson/PrepareArcannaRawJson.py

```diff
@@ -0,0 +1,19 @@
+import demistomock as demisto  # noqa: F401
```

Comment:
When do users should use this script?

Reply:
when they want to send a json string correctly formmatted for Arcanna when  the source comes from elasticsearch for example